satpy/readers/olci_nc.py

```diff
@@ -184,24 +196,40 @@ def _get_solar_flux(self, band):
         return da.map_blocks(self._get_items, d_index.data,
                              solar_flux=solar_flux, dtype=solar_flux.dtype)
 
+    def getbitmask(self, quality_flags, items=None):
+        """Get the quality flags bitmask."""
+        if items is None:
+            items = DEFAULT_L1B_MASK_ITEMS
+
+        bflags = BitFlags(quality_flags, flag_list=L1B_QUALITY_FLAGS)
+
+        return reduce(np.logical_or, [bflags[item] for item in items])
+
     def get_dataset(self, key, info):
         """Load a dataset."""
-        if self.channel != key["name"]:
+        if self.channel is not None and self.channel != key["name"]:
```

Comment:
When is `self.channel` `None`?

Reply:
Thanks for comments. 
When loading `mask` instead of an actual channel, the self.channel=None;  similar as the get_dataset() in the L2 Class at https://github.com/pytroll/satpy/blob/main/satpy/readers/olci_nc.py#L218

Reply:
Ok. It makes me feel like this should be done differently, but this reader code is too big to try to change it now for such a "simple" addition/change. Looks good.